packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -293,6 +300,32 @@ export const Subscriptions = ({
             </div>
           </div>
 
+          {customer.result && showPaymentUpdateForm && (
```

Comment:
TS's typechecker is not good enough to know that `showPaymentUpdateForm` already contains a check on `customer.result`.  🤷‍♂️

Reply:
Hmm, probably a bad smell that could be improved in issue #5683, but :shrug: for now